Userland/Libraries/LibAudio/FlacLoader.cpp

```diff
@@ -344,6 +344,10 @@ ErrorOr<Vector<FixedArray<Sample>>, LoaderError> FlacLoaderPlugin::load_chunks(s
 
     size_t samples_to_read = min(samples_to_read_from_input, remaining_samples);
     Vector<FixedArray<Sample>> frames;
+    // In this case we can know very exactly how many frames we're gonna read.
```

Comment:
```suggestion
    // In this case we can know exactly how many frames we're going to read.
```

Reply:
Huh, I thought I had already formalized this comment.